### PR TITLE
feat(useFileSystemAccess): new function

### PR DESCRIPTION
**Pull request by okxiaoliang4**
create, read and write local files.

### CODE REVIEW

File: packages/core/useFileSystem/index.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { ref } from 'vue-demi'
+
+export function useFileSystem() {
```
Comment (antfu) [inline packages/core/useFileSystem/index.ts:3]: Extend the ConfigureWindows interface to be consistent with others.

File: packages/core/useFileSystem/index.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  const win = window as any
+  const isSupported = win.showSaveFilePicker && win.showOpenFilePicker
+
+  const fileHandle = ref()
+  const content = ref()
```
Comment (antfu) [inline packages/core/useFileSystem/index.ts:8]: Should provide type to them

**Comment by okxiaoliang4** [comment]
ok, is draft now.
maybe i will refactor the code.

TODO:
- [ ] multiple file

File: packages/core/useFileSystemAccess/index.ts [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+}
+
+export type UseFileSystemAccessCommonOptions = Pick<FileSystemAccessShowOpenFileOptions, 'types' | 'excludeAcceptAllOption'>
+export type UseFileSystemAccessShowSaveFileOptions = Pick<FileSystemAccessShowSaveFileOptions, 'suggestedName'>
+
+export type UseFileSystemAccessOptions<T = 'Text' | 'ArrayBuffer' | 'Blob'> = ConfigurableWindow & UseFileSystemAccessCommonOptions & {
+  /**
+   * file data type
+   */
+  dataType?: T
```
Comment (wheatjs) [inline packages/core/useFileSystemAccess/index.ts:87]: Do we maybe want this as a `MaybeRef`? Could there be cases where the type might change?

Comment (okxiaoliang4) [reply packages/core/useFileSystemAccess/index.ts]: Maybe like this?
https://github.com/vueuse/vueuse/pull/1243/commits/e5f602b8fd96a8ef2b5b16d022f5a33a59998ce7#diff-1b72aa40e079e6f5594602eefabee967faf919e43461d36aa28a032f865d2affR172

File: packages/core/useFileSystemAccess/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+category: Browser
+---
+
+# useFileSystemAccess
+
+create and read and write local files[FileSystemAccessAPI](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API)
```
Comment (antfu) [inline packages/core/useFileSystemAccess/index.md:7]:
```suggestion
Create and read and write local files with [FileSystemAccessAPI](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API)
```

**Pull request merged**